.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,19 +28,3 @@ To report a security issue, please email security@diem.com. -->
 ```
 // Paste the output here
 ```
-
-## Expected Behavior
-
-<!-- A clear and concise description of what you expected to happen. -->
-
-## System information
-
-**Please complete the following information:**
-- <!-- Diem Version -->
```

Comment:
This would be good info to have for debugging / reproducing, if we don't have a version, the 0L commit hash of the running code would be good.